File: Python-Task-05/C Files/marks_analyzer.c
#include <stdio.h>

int main() {
    int marks[5], i;
    int highest, lowest, sum = 0;

    printf("Enter marks of 5 students:\n");

    for(i=0;i<5;i++){
        scanf("%d",&marks[i]);
    }

    highest = lowest = marks[0];

    for(i=0;i<5;i++){
        if(marks[i]>highest)
            highest=marks[i];

        if(marks[i]<lowest)
            lowest=marks[i];

        sum += marks[i];
    }

    printf("\nHighest Marks: %d",highest);
    printf("\nLowest Marks: %d",lowest);
    printf("\nAverage Marks: %.2f",(float)sum/5);

    return 0;
}

File: Python-Task-05/Python Files/marks_analyzer.py
marks = []

print("Enter marks of 5 students:")

for i in range(5):
    mark = int(input(f"Student {i+1}: "))
    marks.append(mark)

highest = max(marks)
lowest = min(marks)
average = sum(marks) / len(marks)

print("\n----- Result -----")
print("Highest Marks:", highest)
print("Lowest Marks:", lowest)
print("Average Marks:", average)